.github/workflows/unit.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
     - name: Install Go
       uses: actions/setup-go@v3
       with:
-        go-version: 1.17.x
+        go-version: 1.18.x
```

Comment:
I believe we are on go 1.19 for rancher now. I use 1.19.3. @kinarashah should this be bumped for the operators?

Reply:
+1 for updating to 1.19 if possible